Add overview sections for roles

markweldensmith opened the pull request:
Add two new files reflecting the community and eScience hackweek roles. Text pulled from the website.

Once approved, will add icons from #72 & #71.

markweldensmith committed:
Add overview sections for roles

Add two new files reflecting the community and eScience hackweek roles. Text pulled from the website.

aaarendt committed:
Merge branch 'main' into markweldensmith-hackweek-roles

aaarendt committed:
add roles to TOC

markweldensmith committed:
Fixed out of bounds text.

JessicaS11 commented:
Way back when (pre https://github.com/uwhackweek/hackweeks-as-a-service/pull/48) we had some ideas around how to best include the role content (and some images to try and help folks navigate the roles). How do we envision helping planning teams navigate assigning these roles within their team, and how do the roles fit in with different types of hackweek support we are offering (e.g. those we are actively planning/hosting versus those we are providing only a smaller, consultation participation in)? I'm just noticing that many of the "eScience team" roles are still critical shoes to fill if you're planning an event that we're NOT helping organize, and teams may want to consider having them.

markweldensmith committed:
Add new logos for hackweek roles

markweldensmith committed:
Update _toc.yml

Update directory navigation structure with new file references

markweldensmith committed:
Merge branch 'markweldensmith-hackweek-roles' of https://github.com/uwhackweek/hackweeks-as-a-service into markweldensmith-hackweek-roles

markweldensmith committed:
Added new pages describing hackweek roles

markweldensmith committed:
Changed spacing in _toc.yml

markweldensmith committed:
Added dividing line to bottom of eSci roles page.

markweldensmith committed:
Rename Collaboration-Architect.md to Collaboration-Architect.md

Mark & Jessica figured out a way to change the incorrect directory name in the web browser.

aaarendt commented on docs/_toc.yml:
```diff
@@ -6,6 +6,15 @@ parts:
     - file: overview/training-modules
 
+- caption: Hackweek Roles
+    - file: roles/Community-Lead
```
add ```chapters:``` between lines 9 and 10

aaarendt replied:
There's a mismatch between the ```roles``` and ```Roles``` directory.

aaarendt commented on docs/roles/Collaboration-Architect.md:
````diff
@@ -0,0 +1,24 @@
+```{image} ../../images/Collaboration_Architect.png
````
Fix pathname and image size.

markweldensmith replied:
Jessica caught this in our TB this morning. We were trying to effect the change using the GUI but had no luck. Thanks for fixing.

aaarendt commented on docs/overview/eScience-Staff-Roles.md:
```diff
@@ -0,0 +1,109 @@
+# eScience Staff Team Roles
```
Delete this document?

markweldensmith replied:
If you read the intro I have for this page I figured it might be useful to include a summary of these eSci roles as context for what goes into creating a hackweek.

markweldensmith commented:
@aaarendt It looks like I need to adjust the image size for all the images. Should I re-commit all these pages with the 20% width indicated?

markweldensmith committed:
Remove PNG files

markweldensmith committed:
New SVG logos

markweldensmith committed:
Change code for rendering images

markweldensmith committed:
Removed spaces from TOC

aaarendt committed:
delete

aaarendt committed:
rename with lower case

JessicaS11 commented:
@markweldensmith Is this ready for content review?

As a side note, we'll also need to do some formatting in the .md files to follow best practices for line length. @jomey or @scottyhq I know we've oscillated over time; do either of you recall if we settled on newlines for all new sentences or only after x characters (and what x we used!)? My personal preference is usually a combination (newline for all new sentences/logical breaks like compound sentences AND then any lines that are still above the character limit), because it makes future edits easier to review.

markweldensmith commented:
Yes! Ready for content review.

On Mon, Jan 27, 2025 at 9:02 AM Jessica Scheick ***@***.***>
wrote:

> @markweldensmith
> <https://urldefense.com/v3/__https://github.com/markweldensmith__;!!K-Hz7m0Vt54!itTnI1rSNglv6fvFG4bQ10pfDIBLo_PAwSodagPAfU-yXoHRPRQIvfdlZDXsL6oGeWPP02-LxVOkFOfB0c3la2E$>
> Is this ready for content review?
>
> As a side note, we'll also need to do some formatting in the .md files to
> follow best practices for line length. @jomey
> <https://urldefense.com/v3/__https://github.com/jomey__;!!K-Hz7m0Vt54!itTnI1rSNglv6fvFG4bQ10pfDIBLo_PAwSodagPAfU-yXoHRPRQIvfdlZDXsL6oGeWPP02-LxVOkFOfBkF-aBi0$>
> or @scottyhq
> <https://urldefense.com/v3/__https://github.com/scottyhq__;!!K-Hz7m0Vt54!itTnI1rSNglv6fvFG4bQ10pfDIBLo_PAwSodagPAfU-yXoHRPRQIvfdlZDXsL6oGeWPP02-LxVOkFOfBnKQn6Hg$>
> I know we've oscillated over time; do either of you recall if we settled on
> newlines for all new sentences or only after x characters (and what x we
> used!)? My personal preference is usually a combination (newline for all
> new sentences/logical breaks like compound sentences AND then any lines
> that are still above the character limit), because it makes future edits
> easier to review.
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/uwhackweek/hackweeks-as-a-service/pull/73*issuecomment-2616375587__;Iw!!K-Hz7m0Vt54!itTnI1rSNglv6fvFG4bQ10pfDIBLo_PAwSodagPAfU-yXoHRPRQIvfdlZDXsL6oGeWPP02-LxVOkFOfBTBWeb7o$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/BA3TUVH2QMGG5XZFPXV6CET2MZRARAVCNFSM6AAAAABTJSDAHWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMJWGM3TKNJYG4__;!!K-Hz7m0Vt54!itTnI1rSNglv6fvFG4bQ10pfDIBLo_PAwSodagPAfU-yXoHRPRQIvfdlZDXsL6oGeWPP02-LxVOkFOfB9L2eaxA$>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

jomey commented:
> As a side note, we'll also need to do some formatting in the .md files to follow best practices for line length. @jomey or @scottyhq I know we've oscillated over time; do either of you recall if we settled on newlines for all new sentences or only after x characters (and what x we used!)? My personal preference is usually a combination (newline for all new sentences/logical breaks like compound sentences AND then any lines that are still above the character limit), because it makes future edits easier to review.

I don't remember what/if we had something for this.

Personally, I have a visual guide around 80 characters in all my editors (code of md) that I use the upper limit for characters in one line. Makes it easier for me to use documents in split window view and also edit sentences that are long. I don't do anything around sentences and break up paragraphs with a new line.

Suggest to set a character limit via the `.editorconifg` in the repos that should be picked up by most common IDEs/editors.

aaarendt committed:
Merge branch 'main' into markweldensmith-hackweek-roles

aaarendt commented:
> > As a side note, we'll also need to do some formatting in the .md files to follow best practices for line length. @jomey or @scottyhq I know we've oscillated over time; do either of you recall if we settled on newlines for all new sentences or only after x characters (and what x we used!)? My personal preference is usually a combination (newline for all new sentences/logical breaks like compound sentences AND then any lines that are still above the character limit), because it makes future edits easier to review.
> 
> I don't remember what/if we had something for this.
> 
> Personally, I have a visual guide around 80 characters in all my editors (code of md) that I use the upper limit for characters in one line. Makes it easier for me to use documents in split window view and also edit sentences that are long. I don't do anything around sentences and break up paragraphs with a new line.
> 
> Suggest to set a character limit via the `.editorconifg` in the repos that should be picked up by most common IDEs/editors.

VS Code seems to not recognize the settings in `.editorconfig` which is a bit frustrating. I generally favor Joe's approach and toggle `Ctrl-Z` whenever I need things to wrap for visual purposes. Agreed we should arrive at some standardized approach and suggest we discuss at a future technical meeting.

aaarendt commented:
> Way back when (pre #48) we had some ideas around how to best include the role content (and some images to try and help folks navigate the roles). How do we envision helping planning teams navigate assigning these roles within their team, and how do the roles fit in with different types of hackweek support we are offering (e.g. those we are actively planning/hosting versus those we are providing only a smaller, consultation participation in)? I'm just noticing that many of the "eScience team" roles are still critical shoes to fill if you're planning an event that we're NOT helping organize, and teams may want to consider having them.

@markweldensmith includes [this page](https://deploy-preview-73--hackweek-guidebook.netlify.app/roles/escience-institute-roles) now which I think captures the eScience roles:
